.github/workflows/publish_docker_images.yml

```diff
@@ -62,6 +62,12 @@ jobs:
         run: |
           ./tools/github_workflows/prepare_for_docker_build_and_push.sh ${{ secrets.DOCKERHUB_USERNAME }} ${{ secrets.DOCKERHUB_TOKEN }} asterinas
 
+      - name: Find Asterinas SCML files
+        id: find-aster-scml
+        run: |
+          ASTER_SCML=$(find ./book/src/kernel/linux-compatibility/ -name "*.scml" -type f | sed 's|^\./|/root/asterinas/|' | tr '\n' ' ')
```

Comment:
Do not evaluate the `ASTER_SCML` value at the creation of the Docker image. If more SCML files are added (which is very likely to happen) after the Docker image is released, then these newly-added SCML files won't appear in `ASTER_SCML`. This would be a big surprise to the user.

I think a better approach is to evaluate the SCML file in `/etc/profile` or `/root/.bashrc`:

```
ASTER_SCML=$(find /root/asterinas/book/src/kernel/linux-compatibility/ -name "*.scml" -type f | sed 's|^\./|/root/asterinas/|' | tr '\n' ' ')
```